library/billing.inc

```diff
@@ -216,9 +214,6 @@ function updateClaim($newversion, $patient_id, $encounter_id, $payer_id=-1, $pay
           "pid = '$patient_id' AND encounter = '$encounter_id'");
       }
     }
```

Comment:
TODO note for self. I thought all the GLOBALS['oer_config']['ws_accounting']['enabled'] if/else were removed??